tests/test_annotation.py

```diff
@@ -141,6 +141,9 @@ def test_box_gt_upload(dataset):
     )
 
 
+@pytest.mark.skip(
+    reason="Skip Temporarily - Need to find issue with customObjectIndexingJobId"
+)
 def test_box_gt_upload_embedding(CLIENT, dataset):
```

Comment:
@gatli I am skipping these two tests for now, to unblock CI.

They are consistently failing with `customObjectIndexingJobId` not being in the response.
ie: https://app.circleci.com/pipelines/github/scaleapi/nucleus-python-client/2311/workflows/8c84e971-e6c9-47d3-889a-c7616871f433/jobs/4800/parallel-runs/2

Reply:
https://app.shortcut.com/scaleai/story/629101/an-sdk-test-are-failing-consistently-test-box-gt-upload-embedding

I will investigate this further